android/app/src/main/kotlin/com/example/sloth/AmberSignerPlugin.kt

```
@@ -0,0 +1,524 @@
+                if (it.getColumnIndex("rejected") >= 0) {
+                    return null
+                }
+
+                if (it.moveToFirst()) {
+                    val resultIndex = it.getColumnIndex("result")
+                    if (resultIndex >= 0) {
+                        return it.getString(resultIndex)
+                    }
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)